.github/workflows/pr.yml

```
@@ -0,0 +1,23 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Pull Request CI
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Diese Konfiguration würde PRs bauen: Deine baut alle Branches außer `master`.
```suggestion
on:  
  pull_request:
    branches:
      - master
```